gradle.properties

```diff
@@ -1,4 +1,4 @@
 version=20.10.1-SNAPSHOT
-besuVersion=20.10.0-RC2
+besuVersion=21.1.0-RC2
 distributionIdentifier=besu-plugins
-distributionBaseUrl=https://bintray.com/hyperledger-org/besu-repo/download_file?file_path=
+distributionBaseUrl=https://hyperledger.jfrog.io/artifactory/besu-binaries/besu/
```

Comment:
Should this URL be the Cloudsmith URL?

Reply:
No this link allows pack plugins with Besu . and Besu is not on cloudsmith